CHANGELOG.md

```diff
@@ -73,11 +73,25 @@ Adding a new version? You'll need three changes:
 
 > Release date: TBD
 
+### Added
+
+- Added `--update-status-queue-buffer-size` allowing configuring the size of
+  the status update queue's underlying channels used to buffer updates to the
+  status of Kubernetes resources.
+  [#4267](https://github.com/Kong/kubernetes-ingress-controller/pull/4267)
+
 ### Fixed
 
 - Translator of `GRPCRoute` generates paths without leading `~` when running
   with Kong gateway with version below 3.0.
   [#4238](https://github.com/Kong/kubernetes-ingress-controller/pull/4238)
+- Fixed a bug where the controller sync loop would get stuck when a number of
+  updates for one of Gateway API resources kinds (`HTTPRoute`, `TCPRoute`,
```

Comment:
I believe we missed `GRPCRoute` in the changelog entry

Reply:
Oh, that's true. Fixed it here, will do so on main.